package.json

```diff
@@ -24,7 +24,7 @@
     "lerna": "lerna",
     "build": "lerna run build --scope=@gitgraph/{core,react,js,node} --stream",
     "watch": "lerna run watch --stream",
-    "storybook": "lerna run storybook --stream",
+    "storybook": "sleep 3 && lerna run storybook --stream",
```

Comment:
This may look mysterious in the future, but hopefully this PR description will bring the relevant context behind this 😉

Reply:
yep, that's actually a common problem when using concurrently and still need one task to be completed before the second one